pkgs/tools/misc/urjtag/default.nix

```diff
@@ -0,0 +1,39 @@
+{ pythonBindings ? false
+, jedecSupport ? false
+, python3 ? null
+, stdenv, autoconf, automake, pkgconfig, gettext, intltool, libtool, bison, flex, fetchgit, makeWrapper
+}:
+
+stdenv.mkDerivation rec {
+  name = "urjtag";
```

Comment:
@bjornfor The UrJTAG project does make releases. [Looks like the latest release (0.10) was on April 17th](http://urjtag.org/). Maybe I should just use 0.10 for the version string? It sounds like sha256 hashes are more important and metadata version strings are purely informational (for humans) anyway...?